src/components/Table.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Styled from 'styled-components';
+
+const STable = Styled.div`
```

Comment:
Here as well `TableWrapper` or so. Or `StyledTable`. I would use abbreviations only in rare situations

Reply:
I saw that you use `StyledX` in other places. So naming them here as well like that would be more consistent